plugins/test/unit/plugins/distributors/test_configuration.py

```diff
@@ -149,7 +148,8 @@ def test__is_valid_repo_registry_id(self):
             'upperCase',
             'uppercasE',
             '$ymbols',
-            '$tuff.th@t.m!ght.h@ve.w%!rd.r#g#x.m*anings()'
+            '$tuff.th@t.m!ght.h@ve.w%!rd.r#g#x.m*anings()',
+            '..............'
         ]
```

Comment:
We should also have a string of length 256 and a string with length greater than 256, again to guard against regression in the `_is_valid_repo_registry_id` unit.